docs/source/developapps/scenario.md

```diff
@@ -9,6 +9,17 @@ Fabric, to issue, buy and redeem commercial paper. We're going to use the
 scenario to outline requirements for the development of commercial paper
 applications and smart contracts used by the participant organizations.
 
+## What is a commercial paper?
```

Comment:
Thank you for your contribution @kumaravelrajan.

I have a suggested change for the heading: "What is commercial paper?"

I suggest this because commercial paper is a category of a debt instrument.

Reply:
Thanks for the feedback @denali49. I'll implement the change.